e2e/test/scenarios/sharing/alert/email-alert.cy.spec.js

```diff
@@ -95,3 +130,25 @@ function openAlertForQuestion(id) {
 function toggleChannel(channel) {
   cy.findByText(channel).parent().find("input").click();
 }
+
+function clickAlertBell() {
+  cy.findByTestId("view-footer").within(() => {
+    cy.icon("bell").click();
+  });
+}
+
+function saveAlert() {
+  clickAlertBell();
+
+  cy.findByRole("dialog").within(() => {
+    cy.findByLabelText("Name").type(" alert");
+    cy.findByRole("button", { name: "Save" }).click();
```

Comment:
we have a handy little shortcut for buttons:

https://github.com/metabase/metabase/blob/e21bde8b5d1daba523ed666e06b3ba2956e9690f/e2e/support/commands/ui/button.ts#L32

```suggestion
    cy.button( "Save" ).click();
```